### PR TITLE
Reworked Languages from Main + Removed Eris stuff I missed.

### PR DESCRIPTION
**Pull request by SirSiggles**
## About The Pull Request

Added languages for species' that didn't have any and removed the Eris languages. Also removed Eris stuff I missed.

## Details

<details>
  <summary>Expand</summary>
Added the languages from main Eclipse, except for EAL (To be handled by Nestor in the future.) and removed Jive and Serbian de facto except for on Mercenaries and renamed English Common to Galactic Common.

I also struck out as many things related to Serbian as I could, and cleaned up Moebius and Ironhammer things I missed first time through. and also removed some mentions of NeoTheology for now.
</details>

### CODE REVIEW

**Comment by EvilJackCarver** [comment]
I'm gonna close this for now so that it stops triggering Travis builds/Discord notifications/~~emails~~ when you push a commit.

You have contributor perms, so you should be able to re-open it whenever you're ready.

**Comment by EvilJackCarver** [comment]
WAIT FUCK CONTRIBUTOR != COLLABORATOR

I'M A DUMB

hit a comment here or on discord when it's ready and me or nestor will reopen it for ya

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/structure/closet/secure_closet/personal/scientist
-	name = "moebius scientist's locker"
+	name = "nanotrasen scientist's locker"
```
Comment (EvilJackCarver) [inline code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:2]: We may be able to remove the company name off of the locker for consistency's sake.

File: code/modules/core_implant/cruciform/cruciform.dm [hunk 2]
```diff
@@ -1,9 +1,9 @@
 var/list/disciples = list()
 
 /obj/item/weapon/implant/core_implant/cruciform
-	name = "cruciform"
+	name = "core implant"
 	icon_state = "cruciform_green"
-	desc = "Soul holder for every disciple. With the proper rituals, this can be implanted to induct a new believer into NeoTheology."
+	desc = "Soul holder for anyone who can afford it. With the proper flair, this can be implanted to induct a new believer into Transhumanism."
```
Comment (EvilJackCarver) [inline code/modules/core_implant/cruciform/cruciform.dm:6]: Isn't this the implant necessary for cloning? If so, we may want to look into toning down the religious overtones a bit.

Comment (EvilJackCarver) [reply code/modules/core_implant/cruciform/cruciform.dm:6]: From Nestor 

image

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm (hunk 1 above)
Comment (SirSiggles) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:2]: Yeah, makes sense to me.

**Pull request merged**